linode/manager
# feat: [M3-7366] - Improved Primary Navigation UX

## Description 📝
Overall improvements to primary nav, which includes changes to accommodate longer Cloud Manager feature names.

## Changes  🔄
- Increased nav width to 232px
- Added transition to nav links and adjusted logo transition to match
- Tightened spacing around logos, links, and beta chip according to mocks
- Reduced link font size to .875rem/14px

## Screenshot
| Before      | After |
| ----------- | ----------- |
|![Screenshot 2024-02-07 at 12 35 54 PM](https://github.com/linode/manager/assets/125309814/b1273e8e-9a13-440a-9212-bc93a2aa0c53)|![Screenshot 2024-02-07 at 12 37 02 PM](https://github.com/linode/manager/assets/125309814/bb0bd806-613f-4f11-a6a6-1d096389ac3d)|

## How to test 🧪
- Checkout branch or preview link
- Verify that opening / closing menu works as intended
- Verify that changing breakpoints works as intended
- Verify changes in different browsers
